girder_volview/utils.py

```diff
@@ -5,6 +5,23 @@
 SESSION_ZIP_EXTENSION = ".volview.zip"
 
 
+VIDEO_EXTENSIONS = [
+    ".mp4",
+    ".webm",
+    ".avi",
+    ".mov",
+    ".wmv",
+    ".mpg",
+    ".mpeg",
+    ".mp2",
+    ".ogg",
+    ".flv",
+]
+LARGE_IMAGE_EXTENSIONS = [".svs", ".vms", ".vmu", ".ndpi", ".scn"]  # ".tif", ".tiff",
```

Comment:
This is the list large_image has registered internally:
```
{'extensions': ['bif',
                'czi',
                'db',
                'dcm',
                'dic',
                'dicom',
                'dzi',
                'geotiff',
                'j2k',
                'jp2',
                'jpe',
                'jpeg',
                'jpf',
                'jpg',
                'jpx',
                'json',
                'lif',
                'mrxs',
                'nc',
                'nd2',
                'ndpi',
                'nitf',
                'ntf',
                'ome',
                'ptif',
                'ptiff',
                'qptiff',
                'scn',
                'svs',
                'svslide',
                'tif',
                'tiff',
                'vms',
                'vmu',
                'vrt',
                'vsi',
                'yaml',
                'yml',
                'zarr',
                'zarray',
                'zattrs',
                'zgroup'],
```
Of course, `dcm`, `dic`, and `dicom` shouldn't be included here.

Reply:
I change it to have an inclusive list of extensions VolView can handle.